src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -347,7 +351,9 @@ class DemuxFastqs
  @arg(doc="Name the output files according to the Illumina file name standards.", mutex=Array("illuminaStandards"))
  var illuminaFileNames: Boolean = false,
  @arg(doc="Keep only passing filter reads if true, otherwise keep all reads. Passing filter reads are determined from the comment in the FASTQ header")
- val keepOnlyPassing: Boolean = false
+ val keepOnlyPassing: Boolean = false,
+ @arg(doc="Keep reads marked as control if true, otherwise removes them. Control reads are determined from the comment in the FASTQ header")
+ val keepControlReads: Boolean = true
```

Comment:
1. I'd default this to `false`, to keep the current default behavior of the tool, which is to output all reads. 
2. I word-smithed this to be the same as `--keep-only-passing` for consistency.
```suggestion
 @arg(doc="Keep only reads identified as control if true, otherwise keep all reads. Control reads are determined from the comment in the FASTQ header")
 val keepControlReads: Boolean = false
```